src/classes/Pool.ts

```diff
@@ -386,10 +391,11 @@ export abstract class Pool {
 
     const [, , , , lup] = response[0];
     const [, , , pendingInflator] = response[1];
-    const [debt, collateral, t0np] = response[2];
+    const [debt, collateral, t0np, tp] = response[2];
     const [kickTimestamp] = response[3];
-    const collateralization = debt.gt(0) ? collateral.mul(lup).div(debt) : toWad(1);
-    const tp = collateral.gt(0) ? wdiv(debt, collateral) : BigNumber.from(0);
+    const collateralization = debt.gt(0)
+      ? collateral.mul(lup).div(wmul(debt, COLLATERALIZATION_FACTOR))
```

Comment:
shouldn't we do wdiv instead of .div here?

Reply:
Was done for efficiency, because SDK does not offer a `muldiv` method.  The first `mul` was scaling to 10^36, and the `div` brought it back down to 10^18.  Reworked to use the more expensive operations in 2a978f8c9b4b5d095aab9b6551cdd91c1be84350.